### ISSUE TITLE
Docs for get starting

### ISSUE THREAD

**Issue by tiagoboeing** [prisma/prisma-content-feedback#131]
The [docs here](https://www.prisma.io/docs/get-started/03-build-graphql-servers-with-prisma-TYPESCRIPT-t201/#implement-graphql-api-based-on-crud-building-blocks) contains deprecated commands by [v0.4.1](https://github.com/prisma-labs/nexus-prisma/releases/tag/v0.4.1).

Example:
```
npm install --save nexus graphql nexus-prisma
npm install --save-dev ts-node-dev
```

The latest versions will be installed and `prismaObjectType` or `makePrismaSchema` will not works, specifically in Step 4.

**Comment by Ciboulette** [comment]
I encountered the same issue 😅

**Comment by tammo** [comment]
I could make it work by using the following package versions:

```
"nexus": "0.12.0-beta.14",
"nexus-prisma": "0.3",
```

**Comment by tiagoboeing** [comment]
Yes, in previous versions it works, but the problem is in the step-by-step documentation.  The recommendation is to use the latest version and it is desirable to try the latest features.

See, the commands was deprecated in `nexus-prisma` v0.4.1.
